ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -134,6 +135,57 @@ if tflite_support_is_available
   )
 endif
 
+if tflite2_support_is_available
+  filter_sub_tflite2_sources = ['tensor_filter_tensorflow_lite.cc']
```

Comment:
later work: let's try to reduce duplicated script for tf-lite 1 & 2.

Reply:
Yes. And, we need to talk about the maintenance policy to handle efficiently versions of TF in the near future as we have learned  at the Python package (e.g., Python 2.X , Python 3.X, and Python4.X)